package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "deploy": "gh-pages -d dist",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "server": "node server.js",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Aseadd/ToDo-List.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Aseadd/ToDo-List/issues"
+  },
+  "homepage": "https://github.com/Aseadd/ToDo-List",
+  "devDependencies": {
+    "css-loader": "^6.7.1",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0",
+    "webpack-dev-server": "^4.9.3"
+  },
+  "dependencies": {
+    "gh-pages": "^4.0.0",
+    "lodash": "^4.17.21"
+  }
+}
```

Comment:
Linters `devDependencies` are not installed, follow by step-by-step the [linters guide](https://github.com/microverseinc/linters-config/tree/master/html-css-js).